CHANGES.txt

```diff
@@ -1,4 +1,8 @@
-2.4.0
+2.5.0 (2023-08-01)
```

Comment:
The current version on pypi is version 2.3.0, so I suggest the next version we release will be 2.4.0.